cellcom/src/backend/status.rs

```diff
@@ -0,0 +1,244 @@
+use std::{sync::OnceLock, time::Duration};
+
+use eyre::Result;
+use orb_endpoints::{v2::Endpoints as EndpointsV2, Backend, OrbId};
+use orb_security_utils::reqwest::reqwest::blocking::Client;
+use serde::{Deserialize, Serialize};
+use tracing::debug;
+
+use crate::data::{CellularInfo, WifiNetwork};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct OrbStatusV2 {
+    pub orb_id: Option<String>,
+    pub location_data: Option<LocationDataV2>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct LocationDataV2 {
+    pub wifi: Option<Vec<WifiDataV2>>,
+    pub cell: Option<Vec<CellDataV2>>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct WifiDataV2 {
+    pub ssid: Option<String>,
+    pub bssid: Option<String>,
+    pub signal_strength: Option<i32>,
+    pub channel: Option<u32>,
+    pub signal_to_noise_ratio: Option<i32>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct CellDataV2 {
+    pub mcc: Option<u32>,
+    pub mnc: Option<u32>,
+    pub lac: Option<String>,
+    pub cell_id: Option<u32>,
+}
+
+pub fn client() -> &'static Client {
```

Comment:
@vmenge has been pushing against using this approach, especially in libraries. IMO the client should not be constructed in this helper library, it should be passed in via the configuration for the library (along with the backend type).

This will make for more testable code.